### DISCUSSION TITLE
cypress dev setup [Aug 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3380] [Discord Threads]
> 🙋‍♂️ **Russell**  
## For Cypress in the dev setup, do I just run "npm install" from the root nocodb/ directory?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1010303306024882256/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### yes  
#  
> 🙋‍♂️ **Russell**  
#### Thank you. That works. Do I need to run cypress anytime I’m running backend/frontend? Or only to run tests (like before creating a pull request)?  
#  
> 🙋‍♂️ **wkw**  
#### cypress is for testing only  
#  
> 🙋‍♂️ **wkw**  
#### we perform cypress testing on each pull request as well  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1010303306867953764)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
